src/modules/sync/sync/workflows.ts

```diff
@@ -21,14 +21,16 @@ export const syncWorkflows = async (
     logInfo(logOptions, "standard", "Updating workflows");
 
     await serially(
-      [...operations.updated.keys()].map(codename => () =>
-        modifyWorkflow(
+      [...operations.updated.entries()].filter(([, arr]) => arr.length).map(([key]) => key).map(codename => () => {
```

Comment:
this feels kinda convoluted, don't you think? I see we check whether there are any update operations on line 20, then we perform similar thing on line 24. how about extracting the codenames of workflows to be updated to a separate array, then check its length and if it's >1, perform the actions, otherwise log the "No workflows to update" info? something like this:

```ts
const workflowCodenamesToUpdate = [...operations.updated.entries()].filter(([, arr]) => arr.length).map(([key]) => key);

  if (workflowCodenamesToUpdate.length) {
    logInfo(logOptions, "standard", "Updating workflows");

    await serially(
      workflowCodenamesToUpdate.map(codename => () => { } // rest of the logic
```

Reply:
good catch, updated :)